docs/139-jak-dziala-generowanie-helpa.md

```diff
@@ -0,0 +1,39 @@
+# Jak działa generowanie Helpa  
```

Comment:
brakuje mi krótkiego podsumowania na początku
czyli dlaczego robimy to w ten sposób
i gdzie to jest uruchamiane (że w konfiguracji workflow ustawiamy i takie tam)

Reply:
widze że jest coś o workflow builderze ale ta kolejność mi nie pasuje

powinno być tak że powinno być to w takiej kolejności, w jakiej jest odpalane 
czyli od workflow -> coraz głębiej

Reply:
Teraz może być?